olp-cpp-sdk-core/CMakeLists.txt

```diff
@@ -227,6 +232,11 @@ set(OLP_SDK_CLIENT_SOURCES
     ./src/client/Tokenizer.h
 )
 
+set(OLP_SDK_PARSER_SOURCES
+    ./src/parser/ApiParser.cpp
+    ./src/parser/ApiParser.h
```

Comment:
Please place all near ApiClientLookup for now. We can move them later if we need to.